x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/otel_logs/index.tsx

```diff
@@ -58,16 +59,11 @@ export const OtelLogsPanel: React.FC = () => {
   }, []);
 
   const {
-    services: {
-      share,
-      http,
-      context: { isServerless, stackVersion },
-    },
+    services: { share, http },
   } = useKibana<ObservabilityOnboardingAppServices>();
 
   const AGENT_CDN_BASE_URL = 'artifacts.elastic.co/downloads/beats/elastic-agent';
-  const agentVersion =
-    isServerless && setup ? setup.elasticAgentVersionInfo.agentVersion : stackVersion;
+  const agentVersion = setup?.elasticAgentVersionInfo.agentVersion ?? '';
```

Comment:
Changing this to use the latest EA version which is consistent with other flows that use EA. Relying on `stackVersion` produces an invalid URL on `edge` environments, which are used for E2E tests.

I could not think of the case why we should use `stackVersion` in this specific flow consider that other flows are always using the latest EA, let me know if I've missed something.

Reply:
We did this originally because we weren't sure we wouldn't introduce breaking changes in minor versions for EDOT - we don't for EA, so it's safe to always use the latest available version, but if we don't then we need to pin the EDOT version to the stack version.

With moving towards GA, I'm not sure whether this still applies - @AlexanderWert could you weigh in here?

Reply:
Agreed with @AlexanderWert to discuss this change at the OTel dev sync tomorrow. Will follow up after the discussion.